Algorithm/DP/readme.md

```diff
@@ -41,4 +41,7 @@ be avoided by constructinng a temporary array dp[] and memoizing the computed va
 
 - Longest Common Subsequence ----> [C++](/Code/C++/longest_common_subsequence.cpp)
 
+- Number of times sorted array rotated ---->[C++](/Code/C++/no_of_rotation.cpp)
+
```

Comment:
There is no Dynamic Programming approach in this algorithm. So, please don't link in the DP folder.